livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java

```diff
@@ -60,27 +60,45 @@
 
   public Integer createSession(InterpreterContext context, String kind) throws Exception {
     try {
+      Map<String, String> conf = new HashMap<String, String>();
+      
+      conf.put("spark.master", property.getProperty("zeppelin.livy.master"));
+      
+      conf.put("spark.driver.cores", property.getProperty("spark.driver.cores"));
+      conf.put("spark.executor.cores", property.getProperty("spark.executor.cores"));
+      conf.put("spark.driver.memory", property.getProperty("spark.driver.memory"));
+      conf.put("spark.executor.memory", property.getProperty("spark.executor.memory"));
```

Comment:
@felixcheung Yes ,sure, we should pass all spark.* properties so I did that and made livy interpreter accept any configurations belong to spark.